### ISSUE TITLE
Status table

### ISSUE THREAD

**Issue by martin-lindstrom** [eu-digital-green-certificates/dgc-testdata#19]
Will there be some sort of status table where we that provide test data can get some feedback?

I think that that is pretty important in order to get as many as possible to supply data.

**Comment by SchulzeStTSI** [comment]
What do you mean with "status table"? We create a table like that: 

https://github.com/eu-digital-green-certificates/dgc-testdata/blob/main/BasicDgcTestsList.md

We can add there more columns. Is it that what you have in mind?

**Comment by martin-lindstrom** [comment]
No, not really. As a provider of test data I would like to get feedback from those that have tested against the files. As it is now BG and SE has commented on eachother's pull requests and given the status. SE can parse and validate BG etc.

**Comment by jkiddo** [comment]
Preferable, a github action should run on each commit

**Comment by jkiddo** [comment]
@martin-lindstrom are you guys working on a tool wrapped in docker that can inspect a 'country' folder and the output whether there's conformance/non-conformance?

**Comment by martin-lindstrom** [comment]
@jkiddo I am primarily focused on creating DGC:s. Even though the https://github.com/DIGGSweden/dgc-java can be used to decode and validate DGC:s, this is not my main focus.

What I want is a plain table were the different countries can "communicate results".

BTW, I built a verifier for the test files in this repo: [DGCTestDataVerifier](https://github.com/DIGGSweden/dgc-java/blob/main/create-validate/src/test/java/se/digg/dgc/interop/DGCTestDataVerifier.java).

**Comment by jeroentrappers** [comment]
I have a test program that will iterate through all json files and run out implementation of the certificate validation code.

There are several issues still to be ironed out, however it might be useful for others to take a look at: https://github.com/jeroentrappers/dart_dgc_validate

**Comment by martin-lindstrom** [comment]
@jeroentrappers Yes, that may be useful, but it is not a replacement for a matrix where each and every country apply **their**  validation code on the different test files. As a provider of a test file I typically want to get confirmation that other countries can parse it.

**Comment by jeroentrappers** [comment]
@martin-lindstrom agreed. We are not sure we will finally be able to use the dart implementation, because currently there are some issues with dependencies we use upstream. Otherwise we will fall back to the iOS and Android implementations available. However, these too currently have their issues with the test set in this respository.
